From 7e70fa67e1ae3ecc5e3686ada02dacd4c0852cbf Mon Sep 17 00:00:00 2001
From: Hanna Hawa <hannah@marvell.com>
Date: Thu, 5 May 2016 17:34:14 +0300
Subject: [PATCH 0377/2241] fix: i2c: dts: disable I2C of AP-806 and enable I2C
 of CP-110

Disable I2C of AP-806 and enable I2C of CP-110 because Armada-70x0 used
the MPPs of the AP-806 for SD/eMMC

Change-Id: I58bfc9a5b7032f025a6a7bfe96a0a42600d4b05a
Signed-off-by: Hanna Hawa <hannah@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/29573
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Shadi Ammouri <shadi@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-7040-db-router.dts | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db-router.dts b/arch/arm64/boot/dts/marvell/armada-7040-db-router.dts
index a611093..458cd64 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db-router.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db-router.dts
@@ -64,7 +64,7 @@
 				status = "okay";
 			};
 			i2c@511000 {
-				status = "okay";
+				status = "disabled";
 				clock-frequency = <100000>;
 			};
 		};
@@ -97,7 +97,8 @@
 				status = "okay";
 			};
 			i2c1: i2c@701000 {
-				status = "disabled";
+				status = "okay";
+				clock-frequency = <100000>;
 			};
 			i2c2: i2c@701100 {
 				status = "disabled";
-- 
2.7.4

